### PR TITLE
remove prerelease dependencies

### PR DESCRIPTION
**Pull request by wes-carlson**

### CODE REVIEW

**Review by heliocliu — APPROVED** [review]
please make sure the lock files get updated as well if needed

File: packages/test/end-to-end-tests/package.json [hunk 1]
```diff
@@ -119,41 +119,41 @@
     "moniker": "^0.1.2",
     "nock": "^10.0.1",
     "nyc": "^15.0.0",
-    "old-aqueduct": "npm:@fluidframework/aqueduct@^0.33.0-0",
```
Comment (curtisman) [inline packages/test/end-to-end-tests/package.json:122]: Are there other prerelease dependency in the rest of the client packages still (e.g. dpendency to common-utils and servers?  Should we just update all of it in one change?

Comment (wes-carlson) [reply packages/test/end-to-end-tests/package.json:122]: Not sure, does the script do this?

Comment (curtisman) [reply packages/test/end-to-end-tests/package.json:122]: I don't think I implement a script to automatically do it yet.  probably just want to go thru what you have released and either search and replace, or use bump-version -d

---
In reply to: [563002425](https://github.com/microsoft/FluidFramework/pull/4897#discussion_r563002425) [](ancestors = 563002425)

**Review by curtisman — APPROVED** [review]
:shipit:

**Pull request merged**